website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx

```diff
@@ -36,10 +36,12 @@ description: >-
 
 [JSON API error object]: https://jsonapi.org/format/#error-objects
 
```

Comment:
Howdy! Can you also add a Beta tag to the [Navigation file](https://github.com/hashicorp/terraform-docs-common/blob/c875e000e7d36da4f51896f347819487f867c1fe/website/data/cloud-docs-nav-data.json#L50), by changing the title to: `Audit Trails Tokens <sup>Beta</sup>`?